src/otx/v2/adapters/torch/mmengine/mmaction/model.py

```diff
@@ -0,0 +1,175 @@
+"""OTX adapters.torch.mmengine.mmaction Model APIs."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import copy
+import fnmatch
+import re
+import warnings
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from mmaction.registry import MODELS
+
+from otx.v2.adapters.torch.mmengine.modules.utils.config_utils import CustomConfig as Config
+from otx.v2.api.utils.importing import get_files_dict, get_otx_root_path
+from otx.v2.api.utils.logger import get_logger
+
+if TYPE_CHECKING:
+    import torch
+
+logger = get_logger()
+
+MODEL_CONFIG_PATH = Path(get_otx_root_path()) / "v2/configs/action_classification/models"
+MODEL_CONFIGS = get_files_dict(MODEL_CONFIG_PATH)
+
+def replace_num_classes(d: Config | dict, num_classes: int) -> None:
```

Comment:
Could we move this to a more generic location, maybe, `src/otx/v2/adapters/torch/mmengine` side?

Reply:
This function also used at the `MMSeg/model.py` and I think we could make more general function to replace the num_classes and then used it for all different frameworks. So, how about considering in the future?